packages/next/test/unit/build.test.js

```diff
@@ -0,0 +1,44 @@
+const path = require('node:path');
```

Comment:
The name of the test is generic (= no cron specific), was that a missing test for next?

Reply:
It is sort of a new way to test the next builder. The other unit tests, test some specific functions from the package, while the integration tests create actual Deployments on Vercel. This is sort of a middle ground, where it tests the main build function from the package, but without creating a Deployment.